data-miner/src/index.ts

```diff
@@ -216,7 +229,7 @@ function makeASCTBData(data: any) {
       } 
 
       for(let row of rows) {
-        row.biomarkers = row.biomarkers_gene.concat(row.biomarkers_protein)
+        row.biomarkers = row.biomarkers_gene.concat(row.biomarkers_protein).concat(row.biomarkers_lipids).concat(row.biomarkers_meta).concat(row.biomarkers_prot)
```

Comment:
Use this form instaead:
row.biomarkers = [
   ...row.biomarkers_protein,
   ...row.biomarkers_lipids,
   ...
];

Reply:
Sure Bruce. I will work on it after the meeting.